FRIEND/FRIEND.cpp

```diff
@@ -539,7 +539,7 @@ class FRIEND final : public PluginDelegate
 							if (streq(e->x->helper, "ARM64_SYSREG"))
 							{
 								// fill helper string with system register name
-								auto helper = new char[kMaxElementNameLength];
+								auto helper = new char[kMaxElementNameLength]();
```

Comment:
Good catch! However I believe the more appropriate fix for this issue would be to init buffer inside getSystemRegisterName. Something like `strncpy` instead of first `qstrncat` or even `nameBuffer[0]='\0'`. My point is - function is responsible for the initialization, not the caller.